carta/session.py

```diff
@@ -391,6 +391,31 @@ def append_image(self, path, hdu="", complex=False, expression=ArithmeticExpress
             Arithmetic expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
         """
         return Image.new(self, path, hdu, True, complex, expression)
+    
+    def load_stokes_hypercube(self, output_direcory, directories, files, polarization_type, hdu="", append=False):
+        """Open or append a new Stokes hypercube with the selected Stokes images.
+
+        Parameters
+        ----------
+        output_direcory : {0}
+            The direcory to output the hypercube, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
+        directories : {1}
+            The list of paths to the images, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
+        files : {2}
```

Comment:
can this be a list of path? If the path in the path does not contain `/`, then we load the images from the current directory. If there is `/`, we treat all the paths as absolute path to load images. This would simply the API and improve the UX I believe.

Reply:
Do you mean we combine the directory and the file name together, and then do something like:
![截圖 2023-04-19 下午9 13 39](https://user-images.githubusercontent.com/106953609/233086484-893ce223-4442-4ee4-b7b4-1c0caca3d036.png)
to separate them?